ui/component/splash/view.jsx

```diff
@@ -129,8 +126,12 @@ export default class SplashScreen extends React.PureComponent<Props, State> {
         });
       } else if (this.state.waitingForWallet > MAX_WALLET_WAIT && launchedModal === false && !modal) {
         clearWalletServers();
-        doShowSnackBar(__('The wallet server took a bit too long. Resetting defaults just in case. Shutdown (Cmd/Ctrl+Q) LBRY and restart if this continues.'));
-        this.setState({waitingForWallet: 0});
+        doShowSnackBar(
+          __(
+            'The wallet server took a bit too long. Resetting defaults just in case. Shutdown (Cmd/Ctrl+Q) LBRY and restart if this continues.'
```

Comment:
1. This is a lot of text in a snackbar
1. This fails UI test of explaining how to do something (vs. providing a link to do it).

(But this is also experimental and presumably uncommon, so perhaps not that important to fix.)